src/char/char.c

```diff
@@ -3562,6 +3560,9 @@ static int char_changecharsex(int char_id, int sex)
 	}
 
 	const char *query = "SELECT `account_id`, `class`, `guild_id` FROM `%s` WHERE `char_id`=?";
+	int account_id;
```

Comment:
probably better left 0 for initial value of vars. it will prevent errors if code bellow will use this vars without checking.

Reply:
That's actually impossible. If there's an error while getting the data from the SQL server the function returns.
But I'll re-apply the initial values regardless...